certified-connectors/IA-Connect JML/apiDefinition.swagger.json

```diff
@@ -7022,49 +8550,52 @@
-                  "format": "int32",
-                  "title": "Azure AD groups failed to remove",
-                  "description": "The number of Azure AD groups that the user failed to remove from."
+                "AzureADv2ResetAzureADUserPropertiesResult": {
+                  "type": "boolean",
+                  "title": "Command result",
+                  "description": "The result of the command (success or failure)."
                 },
-                "RemoveAzureADGroupsErrorMessage": {
+                "ErrorMessage": {
                   "type": "string",
-                  "title": "Remove Azure AD groups master error message",
-                  "description": "If the user failed to remove from some of the Azure AD groups, this error message provides details of the problem."
+                  "title": "Error message",
+                  "description": "If the command was not successful, this will contain the error message that was returned."
                 }
               }
             }
           }
         },
-        "operationId": "AzureADv2RemoveUserFromAllGroups"
+        "operationId": "AzureADv2ResetAzureADUserProperties"
       }
     },
-    "/PowerShellAzureADv2/AzureADv2GetAzureADLicenseSKUs": {
+    "/PowerShellAzureADv2/AzureADv2SetAzureADUserManager": {
       "post": {
         "consumes": [
           "application/json"
         ],
         "produces": [
           "application/json"
         ],
-        "summary": "Get Azure AD license SKUs",
-        "description": "Returns a list of Azure Active Directory license Stock Keeping Units (SKUs) which the connected Azure AD is subscribed to, using the Azure AD v2 PowerShell modules.",
+        "summary": "Set Azure AD user's manager",
+        "description": "Set an Azure Active Directory user's mananger.",
         "parameters": [
           {
             "in": "body",
-            "name": "AzureADv2GetAzureADLicenseSKUs",
-            "description": "Inputs for IA-Connect AzureADv2GetAzureADLicenseSKUs action",
+            "name": "AzureADv2SetAzureADUserManager",
+            "description": "Inputs for IA-Connect AzureADv2SetAzureADUserManager action",
             "required": true,
             "schema": {
               "type": "object",
               "properties": {
+                "ObjectId": {
+                  "type": "string",
+                  "title": "User object Id or UPN",
+                  "description": "The Id of an Azure Active Directory user. You can specify a user by UPN (e.g. user@mydomain.onmicrosoft.com) or ObjectId (e.g. 01234567-0123-0123-0123-0123456789ab)."
+                },
+                "Manager": {
+                  "type": "string",
+                  "title": "Manager",
+                  "description": "To add a manager, specify the user's manager Object Id (e.g. a7199172-4c2c-41e5-bed8-bd7a3a1303a5) or UPN (e.g. MyBoss@MyDomain.com). To remove the user's manager, set this field to blank."

```

Comment:
@SimonBondyUltima this description also has a secret-look-a-like-value, different from the previous comment. Please replace it too.